hack/verify-vendor.sh

```diff
@@ -0,0 +1,60 @@
+#!/usr/bin/env bash
```

Comment:
hmm.... this is not being called from `hack/verify-all`, but that script isn't used anywhere as well.... Maybe not in this PR, but we'll have to do some cleanup at some point

Reply:
added to verify-all

Reply:
looks make verify only does linting now, we need to bring back verify-all to `make verify` target

Reply:
yes, I don't think anyone is calling `./hack/verify-all` directly. I was wondering if it being configured via prow though? Have we configured it there? the `./hack/verify-all` script is broken anyways though - the file `./hack/verify-golint` does not even exist.

Reply:
I have fixed `make verify` so that it calls `./hack/verify-all`